### PR TITLE
wait's input is nullable

### PR DESCRIPTION
**Pull request by rgv151**

### CODE REVIEW

File: lib/posix/posix.nim [hunk 1]
```diff
@@ -2316,7 +2316,7 @@ proc timer_settime*(a1: Timer, a2: cint, a3: var Itimerspec,
 proc tzset*() {.importc, header: "<time.h>".}
 
 
-proc wait*(a1: var cint): Pid {.importc, header: "<sys/wait.h>".}
+proc wait*(a1: ptr cint): Pid {.importc, discardable, header: "<sys/wait.h>".}
```
Comment (dom96) [inline lib/posix/posix.nim]: Why ``discardable``?

Comment (rgv151) [inline lib/posix/posix.nim]: because it returns a pid, but almost case we already known it.

Comment (dom96) [inline lib/posix/posix.nim]: It also returns whether it succeeded though, no?

Comment (rgv151) [inline lib/posix/posix.nim]: yep, it always returns child's pid

Comment (dom96) [inline lib/posix/posix.nim]: Alright.

**Pull request merged**